packages/chips/Chip.tsx

```diff
@@ -70,8 +70,8 @@ export class Chip extends React.Component<ChipProps, ChipState> {
   };
 
   state = {
-    classList: new Set(),
-    leadingIconClassList: new Set(),
+    classList: new Set<string>(),
```

Comment:
are these required? Isn't this inferred by ChipState?

Reply:
Turns out that they aren't anymore :-/

Reply:
Ah can't we set the type of the object by doing:

`state: ChipState = {...}`